doc/man3/SSL_CTX_use_certificate.pod

```diff
@@ -158,6 +175,28 @@ is considered public anyway.)
 On success, the functions return 1.
 Otherwise check out the error stack to find out the reason.
 
+For SSL_use_cert_and_key() and SSL_CTX_use_cert_and_key(), the return values are:
+
+=over 4
+
+=item Z<>2
+
+on success of replacing an existing certificate/private key (B<override> is non-0)
+
+=item Z<>1
+
+on success if no existing certificate
+
+=item Z<>0
+
+on failure to replace an existing certificate (B<override> = 0)
+
+=item Z<>-1
```

Comment:
Multiple possible failure codes is annoying in the general case (since our style tends to favor "if (!cal() || !call() || ...)"), though I can see some advantage for application code in having a separate code for "failure to replace".  So I don't think I'll argue for just making all errors return 0, here.

Reply:
Yes, the simple ! option for checking return is nice.

Reply:
If we really don't care about these particular conditions, I can change this to be a boolean return.

Reply:
Looking at #4964, I could just change it to be a boolean, and add an error to the error stack to differentiate failures.